### ISSUE TITLE
Add exit codes 

### ISSUE THREAD

**Issue by Dominik1999** [raiden-network/raiden#5461]
We need to analyze whether we have proper exit codes in Raiden and if not, if we need those. 
The background: Over the weekend some Raiden nodes lost connection to their matrix server and shut down. The Scenario Player was not able to interpret the shutdown correctly. However, if the scenario player would have known the exit codes, we could have see a proper error message. 

- [ ] analyze if there are proper exit codes in Raiden and the services
- [ ] If yes, sit down with @nlsdfnbch and explain those
- [ ] If not, define useful exit codes and implement

**Comment by nlsdfnbch** [comment]
Suggestion @Dominik1999 :
```
- [ ] If exit codes exist, point @nlsdfnbch to their docs - if docs don't exist, supply them
```

**Comment by hackaugusto** [comment]
implemented by 20f8c76050
